docs/en/01_external-broken-links.md

```diff
@@ -0,0 +1,99 @@
+---
+title: External Broken Links Report
+summary: Features of the external broken links report in SilverStripe, how it works, and configuration options
+icon: chain-broken
```

Comment:
That icon doesn't exist in font awesome 5. Did you mean `unlink`?
https://fontawesome.com/v5/search?q=chain-broken&o=r

Reply:
Updated